### DISCUSSION TITLE
Can’t access my Excalidraw file anymore.

### DISCUSSION THREAD

**Discussion by Stam2000** [excalidraw/excalidraw#8966] [General]
The error I encounter occurs when I try to open my Excalidraw file after a close of the application.

[the file](https://drive.google.com/file/d/1IrheV9aOzhsS3_fVONEXsX06g2d5YPum/view?usp=sharing)

this is the error message.
![3ee49d8e3fcf096b19ff91c12550a4794461efad_2_690x445](https://github.com/user-attachments/assets/4c41ee0c-1e33-4a6d-a527-60a81ab73386)

I tried updating my Excalidraw plugin, but without success. I also tried opening the file on another device, but unfortunately, that didn’t work either.

**==:warning: Switch to EXCALIDRAW VIEW in the MORE OPTIONS menu of this document. :warning:== You can decompress Drawing data with the command palette: ‘Decompress current Excalidraw file’. For more info check in plugin settings under ‘Saving’**

I tried following this step, but I couldn’t find the **“Decompress Current Excalidraw File”** functionality anywhere, even after extensive online searches.

**Comment by dwelle** [comment]
Hi, best ask on the Obsidian plugin tracker: https://github.com/zsviczian/obsidian-excalidraw-plugin/issues
